mcs/class/corlib/System.Reflection.Emit/TypeBuilder.cs

```diff
@@ -791,7 +791,7 @@ TypeInfo CreateTypeInfo ()
 					if (!fb.IsStatic && (ft is TypeBuilder) && ft.IsValueType && (ft != this) && is_nested_in (ft)) {
 						TypeBuilder tb = (TypeBuilder)ft;
 						if (!tb.is_created) {
-							AppDomain.CurrentDomain.DoTypeResolve (tb);
+							AppDomain.CurrentDomain.DoTypeBuilderResolve (tb);
```

Comment:
Where is `DoTypeBuilderResolve` defined?

Reply:
Oops, it looks like I forgot to include my `AppDomain.cs` changes.

Reply:
Should be fixed now.